### PR TITLE
Adding accept header for definition get resource

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
Adding accept header for definition get resource

## Issue
fixes https://github.com/wso2/apk/issues/1007

### CODE REVIEW

File: runtime/runtime-domain-service/ballerina/APIClient.bal [hunk 1]
```diff
@@ -36,18 +36,42 @@ import ballerina/crypto;
...
         model:API? api = getAPI(id, organization);
         if api is model:API {
             json|error definition = self.getDefinition(api);
             if definition is json {
                 http:Response response = new;
-                response.setJsonPayload(definition);
-                response.statusCode = 200;
-                return response;
+                if accept is string {
+                    if accept == "application/json" || accept == "*/*" {
```
Comment (tharindu1st) [inline runtime/runtime-domain-service/ballerina/APIClient.bal:50]: use constants for media types

File: runtime/runtime-domain-service/ballerina/APIClient.bal [hunk 2]
```diff
@@ -36,18 +36,42 @@ import ballerina/crypto;
...
+                        response.setJsonPayload(definition);
+                        response.statusCode = 200;
+                        return response;
+                    } else if accept == "application/yaml" {
```
Comment (tharindu1st) [inline runtime/runtime-domain-service/ballerina/APIClient.bal:54]: same goes here

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: runtime/runtime-domain-service/ballerina/APIClient.bal (hunk 1 above)
Comment (CrowleyRajapakse) [reply runtime/runtime-domain-service/ballerina/APIClient.bal]: fixed and resolved

File: runtime/runtime-domain-service/ballerina/APIClient.bal (hunk 2 above)
Comment (CrowleyRajapakse) [reply runtime/runtime-domain-service/ballerina/APIClient.bal]: fixed and resolved

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**